diff --git a/linux/installer/common/sgx-aesm-service/Makefile b/linux/installer/common/sgx-aesm-service/Makefile
index 71face18..c86ca03f 100644
--- a/linux/installer/common/sgx-aesm-service/Makefile
+++ b/linux/installer/common/sgx-aesm-service/Makefile
@@ -42,14 +42,10 @@ AESMD_CONF_DEL=$(if $(wildcard /run/systemd/system/.*),aesmd.conf,$(if $(wildcar
 AESMD_CONF_PATH=$(if $(wildcard /run/systemd/system/.*),$(if $(wildcard /lib/systemd/system/.*),/lib/systemd/system,/usr/lib/systemd/system),$(if $(wildcard /etc/init/.*),/etc/init/))
 
 ifeq ($(AESMD_CONF_NAME),)
-ifneq ($(shell awk -F/ '$$2 == "docker"' /proc/self/cgroup),)
 AESMD_CONF_NAME=aesmd.service
 AESMD_CONF_DEL=aesmd.conf
 AESMD_CONF_PATH=/lib/systemd/system
 $(warning "You may need to start aesmd manually after it's installed!")
-else
-$(error "Unsupported platform - neither systemctl nor initctl is found!")
-endif
 endif
 
 QE_VER=1.0.0
diff --git a/linux/installer/deb/local_repo_tool/conf/distributions b/linux/installer/deb/local_repo_tool/conf/distributions
index 2f9137fc..b7f6ddf6 100644
--- a/linux/installer/deb/local_repo_tool/conf/distributions
+++ b/linux/installer/deb/local_repo_tool/conf/distributions
@@ -29,3 +29,11 @@ Architectures: amd64
 Components: main
 Description: ubuntu/hirsute repository for SGX PSW
 DebIndices: Packages .
+
+Origin: Intel Corporation
+Label: Intel Corporation
+Codename: impish
+Architectures: amd64
+Components: main
+Description: ubuntu/impish repository for SGX PSW
+DebIndices: Packages .
